code-challenges.js

```diff
@@ -30,26 +41,39 @@ const temperature2 = 350
 
 const temperature3 = 212
 // Expected output: "212 is at boiling point"
-
+console.log(boilingPoint(temperature1));
+console.log(boilingPoint(temperature2));
+console.log(boilingPoint(temperature3));
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
-// Pseudo code:
-
+// Pseudo code: create a function named combineAndGetLenght. It takes two arrays as arguments name them arra1 and arra2. the function will create a variable named combinedArray that stores an array and uses the .concat method to combine the two arrays passed as argument. return the lenght the new array using the .lenght method
+const combineAndGetLength =(array1, array2)=> {
+    let combinedArray = array1.concat(array2);
+    return combinedArray.length;
+  }
+  
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+
+console.log(combineAndGetLength(padres1984WorldSeriesRuns,padres1998WorldSeriesRuns));
 // Expected output: 9
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
-// Pseudo code:
+// Pseudo code: create a function named reserveseString. Passed a string as argumen. the function crates a variable name reversedString. this variable will storsed the array passed as argumnet using the strings methods to split each letter, reverse them and then join them. also return the new variable.
```

Comment:
Nice job chaining built in methods. Keep in mind you can just return everything instead of creating a variable and returning the variable